src/github/client/github-installation-client.ts

```diff
@@ -481,7 +481,8 @@ export class GitHubInstallationClient extends GitHubClient {
 	 * Get a file at a given path from a repository.
 	 * Returns null if the file does not exist.
 	 */
-	public async getRepositoryFile(owner: string, repo: string, path: string): Promise<string | undefined> {
+
+	public getRepositoryFile = async (owner: string, repo: string, path: string): Promise<string | undefined> => {
```

Comment:
Modified this as tests weren't identifying this as a method on the installation client.